src/assets/scss/components/search.scss

```diff
@@ -0,0 +1,72 @@
+.search-input {
+  border: 0;
+  border-bottom: 2px solid rgb(51, 51, 51);
+  border-radius: 0;
+  justify-content: space-between;
+  outline: none;
+  padding: 0px 10px;
+  padding-left: 0px;
+  position: relative;
+  width: 290px;
+}
+
+.search-input .filter-title {
+  display: none;
+}
+
+.search-input .dropdown-container .btn {
+  height: 30px;
+}
+
+.search-input .dropdown-container .eos-icons {
+  padding: 0px;
+}
+
+.search-input input {
+  border: 0px;
+  outline: none;
+  padding: 0px;
+  width: 147px;
+}
+
+.user-suggest-dropdown .dropdown-list {
+  width: 100%;
+}
+
+.user-suggest-dropdown {
+  background-color: #ffffff;
+  border-radius: 10px 0px 10px 10px;
+  height: auto;
+  max-height: 160px;
+  overflow-y: scroll;
+  position: absolute;
```

Comment:
Please make sure that absolute positioning isn't causing issues on resizing the browser window. @sharmaaditya570191 if you test the functionality please go through this part.

Reply:
I have checked it.